### DISCUSSION TITLE
question: Infinite scroll

### DISCUSSION THREAD

**Discussion by alehkudrevich** [table-library/react-table-library#61] [Q&A]
Is there any way to implement Infinite Scroll functionality? With loading data from server.

**Comment by rwieruch** [comment]
Tbh I didn't look into this yet. It's possible, but one would have to come up with the implementation. So far I have only implemented fetching on demand in the storybook (see Server stories).

**Reply by gofford** [reply]
Any progress on an example of infinite scroll? The storybook suggests that it's a WIP so I'm curious to see if this has changed before I look into it myself.

**Reply by lucastobrazil** [reply]
+1
